commit 813bde765bbcf7643f71749e1d95025459cb9d48
Author: Jookia <contact@jookia.org>
Date:   2021-08-27 08:10:25 +1000

    brcmfmac: Increase PMU_MAX_TRANSITION_DLY to 5 seconds
    
    In some cases the AP6236 will take around 3120ms to set the clock,
    causing this error:
    
    brcmfmac: brcmf_sdio_htclk: HT Avail timeout (1000000): clkctl 0x50
    brcmfmac: brcmf_sdio_htclk: HT Avail timeout (1000000): clkctl 0x50
    
    Wait a little longer before timing out.
    
    This usually happens after cutting power unexpectedly.

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
index 5fe01048b18c..dc952168f23d 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
@@ -310,7 +310,7 @@ struct rte_console {
  * max out at a 1 second (1000000us).
  */
 #undef PMU_MAX_TRANSITION_DLY
-#define PMU_MAX_TRANSITION_DLY 1000000
+#define PMU_MAX_TRANSITION_DLY 5000000
 
 /* Value for ChipClockCSR during initial setup */
 #define BRCMF_INIT_CLKCTL1	(SBSDIO_FORCE_HW_CLKREQ_OFF |	\
